packages/analytics/amplify_analytics_pinpoint/example/android/.idea/codeStyles/Project.xml

```diff
@@ -1,10 +1,25 @@
 <component name="ProjectCodeStyleConfiguration">
```

Comment:
Same as previous... I think there might be an issue with your gitignore?

Update: actually just noticed that these were always present in analytics? In any case, I think they can be removed.